scripts/kind-e2e/e2e.sh

```diff
@@ -77,7 +77,7 @@ function setup_broker() {
         echo Installing broker on cluster1.
          sd=
          [[ $lighthouse = true ]] && sd=--service-discovery
-         ../bin/subctl --kubeconfig ${PRJ_ROOT}/output/kind-config/dapper/kind-config-cluster1 deploy-broker ${sd} |& cat
+         ../bin/subctl --kubeconfig ${PRJ_ROOT}/output/kind-config/dapper/kind-config-cluster1 deploy-broker --service-discovery ${sd} |& cat
```

Comment:
This is not needed. $sd is set to service discovery in previous like already if lighthouse is true.

Reply:
Done.